README.md

```diff
@@ -13,5 +13,98 @@ Most of performed methods are logged using NLog, so you can easily see a history
 
+### Quick start
+
+1. To start work with this package, simply add the nuget dependency Aquality.WinAppDriver to your project.
+
+2. Configure path to your application executable at settings.json:
+ - Create a `Resources` folder in your project and copy [settings.json](Aquality.WinAppDriver/src/Aquality.WinAppDriver/Resources/settings.json) into it. 
+ - Select `Copy to Output Directory`: `Copy always` option at settings.json file properties.
+ - Open settings.json and find `applicationPath` option under the `driverSettings` section. Replace the value with the path to your app.
+
+3. Ensure that [WinAppDriver](https://github.com/microsoft/WinAppDriver) is set up at your machine where the code would be executed.
+
+4. Start WinAppDriver service before your code execution. There are several options to do this:
+ - Start WinAppDriver.exe via external tool or command (e.g. manually with cmd/PS from the installation directory; via [task for you CI](https://marketplace.visualstudio.com/items?itemName=WinAppDriver.winappdriver-pipelines-task) )
+ - Start WinAppDriver automatically via AppiumLocalService - this option is integrated in our package, just set `"isRemote"` to `false` at your settings.json. This option requires node.js to be preinstalled on your machine.
+ - Start WinAppDriver.exe manually calling the method `AqualityServices.WinAppDriverLauncher.StartWinAppDriverIfRequired();` from your code.
+ 
+5. (optional) Launch an application directly by calling `AqualityServices.Application.Launch();`. 
+
+Note: If you don't start an Application directly, it would be started with the first call of any Aquality service or class requiring to interact with the Application.
+
+6. That's it! Now you can work with Application via AqualityServices or via element services.
+```csharp
+
+ AqualityServices.Application.MouseActions.Click();
+ AqualityServices.Application.KeyboardActions.SendKeysWithKeyHold("n", ModifierKey.Control);
+```
+
+7. To interact with Application's windows, forms and elements, we recommend to follow the PageObjects pattern. This approach is fully integrated into our package.
+To start with that, you will need to create a separate class for each window/form of your application, and inherit this class from the [Window]() or [Form](https://github.com/aquality-automation/aquality-winappdriver-dotnet/blob/master/Aquality.WinAppDriver/src/Aquality.WinAppDriver/Forms/Form.cs) respectively. 
+
+
+Notice: 
+ - By "Form" we mean part of the main window of your application (by default) or part of the any other window (specified as a constuctor parameter). An example of form is a MenuBar or some section.
+ - By "Window" we mean the separate window of your application (would be searched from the [RootSession](https://github.com/microsoft/WinAppDriver/wiki/Frequently-Asked-Questions#when-and-how-to-create-a-desktop-session) by default). In other words, the "Window" is a form which is separate from the main application window.
+
```

Comment:
What do you think if leave only one entity - Form or Window? As an end user I don't want to think about it, just have one base class for form/window. Btw, window is also form, isn't it?

Reply:
You will use Form in most of the cases, yes.
Yes, Window is a Form separate from the main application window. But the mechanism of the search from the root session is quite challenging. From the other
There are few cases where you should use Window class instead of Form:
- Your application consists of several Windows;
- You need to interact with more than one application;
- You need to perform actions before/after the main application run (e.g. reinstall, update);